commit 24ac14da6b7b3c5d91dca68c97c8f7ccd5481748
Author: Poul-Henning Kamp <phk@FreeBSD.org>
Date:   Fri Feb 23 07:57:12 2018 +0000

    EPIPE is a documented errno in tcp(7) on linux
    
    Fixes: #2582

WMF Bug: T224694

diff --git a/lib/libvarnish/vtcp.c b/lib/libvarnish/vtcp.c
index 79498c41c..af8a147ab 100644
--- a/lib/libvarnish/vtcp.c
+++ b/lib/libvarnish/vtcp.c
@@ -569,7 +569,7 @@ VTCP_Check(int a)
 {
 	if (a == 0)
 		return (1);
-	if (errno == ECONNRESET || errno == ENOTCONN)
+	if (errno == ECONNRESET || errno == ENOTCONN || errno == EPIPE)
 		return (1);
 #if (defined (__SVR4) && defined (__sun)) || defined (__NetBSD__)
 	/*
